src/main/cpp/subsystems/LedSubsystem.cpp

```diff
@@ -23,9 +23,9 @@ frc2::CommandPtr LedSubsystem::ShowFromState(StateGetter stateGetter) {
             return ScoringSpeaker();
         case RobotState::ScoringAmp:
             return ScoringAmp();
-        case RobotState::Stowing:
-            return Stowing();
-        case RobotState::Idling:
+        case RobotState::Loaded:
+            return Loaded();
+        case RobotState::Manual:
             return Idling();
```

Comment:
Shouldn't this return `Manual();`?

Reply:
No since Idling is simply a passive animation and could be used in multiple areas